cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
```

Comment:
Are there prerequisites a user must have before running this? What assumptions are you making? You don't necessarily have to tell them how to do everything but I would say things like
- What version of Python is required
- What version of Composer you can use this on
- Any IAM permissions they would need on their project/Composer environment
- Any restrictions (can you run this across all regions and zones that Composer is available in? Can you run this with all networking configuirations?)

Reply:
Good point I have added a Prerequisites section to cover these points, thanks !